bolt/lib/Passes/PAuthGadgetScanner.cpp

```diff
@@ -355,37 +399,39 @@ class PacRetAnalysis
       return State();
     }
 
+    // First, compute various properties of the instruction, taking the state
+    // before its execution into account, if necessary.
+
+    BitVector Clobbered = getClobberedRegs(Point);
+    // Compute the set of registers that can be considered as written by
+    // an authentication instruction. This includes operations that are
+    // *strictly better* than authentication, such as materializing a
+    // PC-relative constant.
```

Comment:
See my comment above, I hope we can come up with a better name than `AuthenticatedOrBetter`.